daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -308,9 +308,7 @@ runCommand = \case
     RunJar {..} ->
         (if shutdownStdinClose then withCloseOnStdin else id) $
         runJar jarPath mbLogbackConfig remainingArguments
-    New {..}
-        | templateNameM == Just "create-daml-app" -> runCreateDamlApp targetFolder
-        | otherwise -> runNew targetFolder templateNameM [] []
+    New {..} -> runNew targetFolder templateNameM [] []
     CreateDamlApp{..} -> runCreateDamlApp targetFolder
```

Comment:
`create-daml-app foobar` now does something different from `new foobar create-daml-app`. We should either kill the command `create-daml-app` or make it do the same.

Reply:
Planning to kill the `create-daml-app` command and just use `daml new`. I was keeping it in for now in case we still want to fetch from the repo in a transition period or something

Reply:
No need to keep backwards-compatibility for a feature that's never been released, even in snapshots.

Reply:
I made `daml create-daml-app foo` an alias for `daml new foo create-daml-app` for now.

Reply:
Will need to figure out what to do with the helper tests though

Reply:
Just kill the helper tests. They don’t make any sense now. As long as we have an integration test that checks that at least `daml new` and `daml build` works, we should be good.